### DISCUSSION TITLE
Add an option to keep native IDE suggestions

### DISCUSSION THREAD

**Discussion by ByteZ1337** [community/community#8444] [Copilot Conversations] [answered]
Hello, I'm currently using Copilot with IntelliJ Ultimate and I love it. However, sometimes Copilot suggests code that is only correct for the first line or part of a line.
For example:
I need:
```kotlin
val newClass = ClassWrapper(name)
```
but Copilot suggests:
```kotlin
val newClass = ClassNode()
```
Which means I won't get any native IDE suggestions until I finish typing ``val newClass = ClassW``. After a while, this leads to a lot of wasted time and frustration. An option to keep native IDE suggestions while copilot is giving suggestions would fix this problem. Would it be possible to add such an option in one of the next updates?

**Comment by jansorg** [comment, marked as answer]
@ByteZ1337 The next update will contain an (experimental) setting to show both Copilot's and the IDE's completions. We'd appreciate your feedback!

**Reply by ByteZ1337** [reply]
Sounds good!

**Reply by jansorg** [reply]
1.1.3 has been released and contains the new setting. It should work okay, but there may be a few edge-cases which need improvements. Your feedback is appreciated!

**Reply by ByteZ1337** [reply]
Just tested it a bit, and I found a little bug: Sometimes the handwritten code is being rendered on the right of copilots suggestion:
![u9bOkda](https://user-images.githubusercontent.com/42820438/144876055-46a202fe-8f58-4fcb-9f4c-c3da4deb1955.png)
I'm not sure if this is even related to the IDE suggestions but everything else works great! Thanks for the update :)

**Reply by jansorg** [reply]
Thanks! We'll track and fix this internally.
